eng/targets/Helix.props

```diff
@@ -44,4 +45,9 @@
     <HelixPreCommand Include="call RunPowershell.cmd InstallNode.ps1 $(NodeVersion) %25HELIX_CORRELATION_PAYLOAD%25\node\bin || exit /b 1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TestDependsOnYarn)' == 'true' AND '$(IsWindowsHelixQueue)' == 'true'">
+    <HelixPreCommand Include="call RunPowershell.cmd InstallJdk.ps1 $(JdkVersion) || exit /b 1" />
```

Comment:
Split these into two different groups. One for yarn and one for java.

Reply:
Good idea